samples/RaspberryPiInstructions.md

```diff
@@ -1,70 +1,104 @@
-# .NET Core on Raspberry Pi
+# Run .NET Core Apps on Raspberry Pi
 
-Arm32 builds are available as community supported builds for .NET Core 2.0. 
-**There is no SDK that runs on ARM32** but you can publish an application that will run on a Raspberry Pi. 
+.NET Core builds are available that run on the Raspberry Pi and similar hardware. In particular, the Raspberry Pi uses an ARM chip, and requires .NET Core ARM builds. .NET Core supports ARM32 and ARM64.
 
-These steps have been tested on a RPi 2 and RPi 3 with Linux and Windows.
+You can build an application for ARM either on an ARM device or on your X64 machine. Both options are documented below.
 
-Note: All models of generation 1 and Pi Zero are not supported because the .NET Core JIT depends on armv7 instructions not available on those versions.
+We recommend that you use .NET Core 3 for Raspberry Pi development. There are significant improvements in .NET Core 3 that make the experience much better.
 
-## Creating an app:
+Note: .NET Core supports Raspberry Pi 2 and Pi 3. We recommend the Pi 3.
 
-* [Install .NET Core SDK](https://www.microsoft.com/net/core) into a supported developer configuration.
-(Raspberry Pi itself is supported only as deployment target but there is an unsupported version of the SDK available as well.)
+Note: You can run [32-bit Linux](https://www.raspberrypi.org/downloads/raspbian/), [64-bit Linux](https://gist.github.com/richlander/467813274cea8abc624553ee72b28213#how-to-install-arm64-builds) and [32-bit Windows](https://docs.microsoft.com/en-us/windows/iot-core/downloads) on the Pi. Any of those options will work with .NET Core. On Linux, it may be helpful to [Starting ssh automatically at boot time](https://raspberrypi.stackexchange.com/questions/1747/starting-ssh-automatically-at-boot-time).
```

Comment:
> [Starting ssh automatically at boot time](https://raspberrypi.stackexchange.com/questions/1747/starting-ssh-automatically-at-boot-time).

NIT: Should that be instead:
`[start ssh automatically at boot time](https://raspberrypi.stackexchange.com/questions/1747/starting-ssh-automatically-at-boot-time).`